README.md

```diff
@@ -57,6 +57,7 @@ docker-compose up
 1. Then
 ``./buildrun.ps1``
 
+(if you are having issues with Docker not starting up, try uninstalling and installing the older version: https://download.docker.com/win/stable/14687/Docker%20for%20Windows%20Installer.exe  ref: https://github.com/docker/for-win/issues/1514)
```

Comment:
I'd suggest adding a header for a "Troubleshooting" subsection and linking text, as opposed to raw links.

## Example
```
### Troubleshooting
If you are having issues with Docker not starting up, try uninstalling and installing the older version.

 [Docker Windows Older Version Installer](https://download.docker.com/win/stable/14687/Docker%20for%20Windows%20Installer.exe)  
Relevant Docker Issue: [Upgrade then Unable to create File System Image · Issue #1514 · docker/for-win](https://github.com/docker/for-win/issues/1514)
```

## Rendered as
### Troubleshooting
If you are having issues with Docker not starting up, try uninstalling and installing the older version.

 [Docker Windows Older Version Installer](https://download.docker.com/win/stable/14687/Docker%20for%20Windows%20Installer.exe)  
Relevant Docker Issue: [Upgrade then Unable to create File System Image · Issue #1514 · docker/for-win](https://github.com/docker/for-win/issues/1514)